trufflescripts/start_auction.js

```diff
@@ -84,7 +85,7 @@ module.exports = async () => {
   if (argv.a) account = argv.a
   else if (argv.buyer) {
     [, account] = accounts
-  } else if (argv.seller) {
+  } else if (argv.seller || argv.master) {
     [account] = accounts
```

Comment:
what's the use of `--master` if it doesn't change seller?

Reply:
consistency, others use master flag